emgapi/migrations/0010_run_assembly_split.py

```diff
@@ -88,29 +88,30 @@ def populate_assemblies(apps, schema_editor):
         if AnalysisJob.objects.filter(run=run).count() < 1:
             run.delete()
 
-# def delete_duplicated(apps, schema_editor):
-#     Run = apps.get_model("emgapi", "Run")
-#     ExperimentType = apps.get_model("emgapi", "ExperimentType")
-#     experiment_type = ExperimentType.objects.get(experiment_type="assembly")
-#
-#     for run in Run.objects.filter(experiment_type=experiment_type):
-#         run.delete()
 
+def add_experiment_types(apps, schema_editor):
```

Comment:
could you please move it to client fixture?
by that migration experiment types has to be in the database